homeassistant/components/camera/nest.py

```diff
@@ -76,7 +78,34 @@ def brand(self):
         """Return the brand of the camera."""
         return NEST_BRAND
 
-    # This doesn't seem to be getting called regularly, for some reason
+    @property
+    def supported_features(self):
+        """Nest Cam support turn on and off."""
+        return SUPPORT_ON_OFF
+
+    @property
+    def is_on(self):
+        """Return true if on."""
+        return self._online and self._is_streaming
+
+    def turn_off(self):
+        """Turn off camera."""
+        _LOGGER.debug('Turn off camera %s', self._name)
+        # Calling Nest API in is_streaming setter
+        self.device.is_streaming = False
+        self.schedule_update_ha_state(True)
+
+    def turn_on(self):
+        """Turn on camera."""
+        if not self._online:
+            raise HomeAssistantError('Camera {} is offline.'
```

Comment:
We usually don't raise within entity actuator methods. Log error and return is normal.